src/state/shellscripts/sandbox_exec.sh

```diff
@@ -20,10 +20,10 @@ add_mounts() {
 }
 
 if [ -z ${TMPDIR+x} ]; then
-  # Others applications obtain the per-user temporary
+  # Other applications obtain the per-user temporary
   # directory differently; the latter should be made readable/writable
   # too and getconf seems to be a robust way to get it
-  if [ -z /usr/bin/getconf ]; then
+  if [ -n "$(which getconf)" ]; then
```

Comment:
Using the `command` builtin is a bit faster and more future proof (in case Apple wants to remove `which`). It is also what is used in the rest of the file (see line 38 for example)
```suggestion
  if command -v getconf > /dev/null; then
```

Reply:
Oh that's what `command` does!
Thanks for the tip :)
Fixed